src/payment/index.js

```diff
@@ -1,3 +1,33 @@
+import { getConfig } from '@edx/frontend-platform';
+
+
+// HACK: short-lived function to send some testing events; sticking it here since there doesn't
+// seem to be a general 'utils' file
+// HACK: deliberately not doing this through the regular sendTrackEvent system to more closely
+// match how these will be sent in the static page test
+/* eslint-disable no-param-reassign */
+/* eslint-disable prefer-template */
+/* eslint-disable dot-notation */
+export function sendRev1074Event(eventType, eventData) {
+  eventData['_export'] = 'false'; // Don't let these events be exported to partners
+  if (window.performance.timing) {
+    eventData.timing = window.performance.timing.toJSON();
+  }
+  const encodedEvent = [
+    'event_type=edx.experiment.rev1074.' + encodeURIComponent(eventType),
+    'page=' + encodeURIComponent(window.location.href),
```

Comment:
why do we need to encode the different variables? is this for security or something

Reply:
As far as I could tell, the GET does _not_ encode them automatically, please let me know if there's a better way.  (Note: this is just encoding, not encrypting, no security implications around this)

Reply:
why is the encoding necessary in that case? if not necessary we could save some characters on the static page

Reply:
Ooohhh, you mean _that_ specific instance; probably not?  (The "spec" is _very_ unclear about what needs to be encoded in different cases)  I'll try without encoding that one 👍

Reply:
Dropped the encode calls on things that don't really need it.